### ISSUE TITLE
Is it possible to change the applet's position relative to the other icons on the right side?

### ISSUE THREAD

**Issue by tedlavarias** [paradoxxxzero/gnome-shell-system-monitor-applet#629]
Hi there! I love this extension! I'm just hoping there's an easy way to reorder its position relative to other extension icons when located on the right side of the top bar? I prefer to keep my clock in the middle of the top bar (respecting the default Gnome setup) and would like the system-monitor extension to display in the right section of the top bar, but on the left side of all the other indicator icons. I've tried manually manipulating the numeric value (line 1132, 3rd parameter) in the .addToStatusArea() function, but that didn't work for me and gave me some very strange output! Like having 5 or 6 copies of the CPU graph displayed simultaneously! I just reset the extension's settings in dconf-editor to get back to their defaults though, but it would be nice to keep all of my indicator icons grouped together. Thanks again for the awesome extension!

**Comment by rkochar** [comment]
@tedlavarias, yes it is. I'm not a maintainer of this project but check [this](https://askubuntu.com/questions/453969/how-can-i-order-gnome3-shell-extensions-at-the-top) link.

**Comment by rkochar** [comment]
Update: The old tried and tested reboot worked. idk why, but the ordering has fixed itself for me.
